strongbox-resources/strongbox-storage-resources/strongbox-storage-api-resources/src/main/resources/etc/conf/strongbox.xml

```diff
@@ -188,7 +208,14 @@
                         <repository>storage-springsource-proxies:springsource-proxies</repository>
                         <repository>storage-ivy-proxies:group-ivy-proxies</repository>
                         <repository>storage-sbt-proxies:group-sbt-proxies</repository>
-                        <repository>storage-third-party:third-party</repository>
+                        <repository>storage-third-party:nuget-third-party</repository>
```

Comment:
This is incorrect. There needs to be a separate group repository for NuGet repositories. Group repositories are supposed to be per layout -- you shouldn't have a group repository that contains both Maven and NuGet repositories, 
as this will cause a big resolution mess at some point.

If you have a look at the way I have structured it for Maven, you might notice that I have:
* Storages for hosted repositories (only)
* Storages for proxies (only)
* Where appropriate, I've added group repositories

Could you please create a new group repository for this, perhaps under a new storage?

Thanks! :)

Reply:
yes, it's already in `nuget-public` group. This was just copypaste :)

But do you really want to move all nuget repositories in separate storages? I think that it looks fine to use `public` and `storage-third-party` for Nuget repositories too.